package.json

```diff
@@ -36,6 +36,7 @@
     "eslint": "^8",
     "eslint-config-next": "14.2.15",
     "jsdom": "^25.0.1",
+    "react-router-dom": "^7.1.0",
```

Comment:
Wozu benötigen wir diese Dependency? Das Routing wird mittels next gemacht.

Reply:
weil ich in der button funktion kein routing mit href:"pfad/ziel" machen kann. da ist das mit dem router object.

Reply:
nimm ich raus